### PR TITLE
Function speedup

### PR DESCRIPTION
**Pull request by andLaing**
Some clarification and performance enhancement of the functions

### CODE REVIEW

File: antea/utils/map_functions.py [hunk 1]
```diff
@@ -227,7 +227,6 @@ def load_corrections(filename, *,
     dst  = load_dst(filename, group, node)
     x, y = np.unique(dst[x_name].values), np.unique(dst[y_name].values)
     f, u = dst[f_name].values, dst[u_name].values
-    n    = dst[n_name].values
```
Comment (paolafer) [inline antea/utils/map_functions.py:230]: I guess that `n` can be removed from the arguments of the function, too.

Comment (andLaing) [reply antea/utils/map_functions.py:230]: From where?

Comment (andLaing) [reply antea/utils/map_functions.py:230]: You mean `n_name`?

Comment (paolafer) [reply antea/utils/map_functions.py:230]: > From where?

From the arguments of the `load_corrections` function, since it is not used in the function.

Comment (paolafer) [reply antea/utils/map_functions.py:230]: > You mean `n_name`?

Yes.

Comment (andLaing) [reply antea/utils/map_functions.py:230]: Done.

I couldn't find any usage of that parameter and the tests pass in my local.

**Review by paolafer — APPROVED** [review]
This PR simplifies several functions, improving the code readability and performance. Approved!

**Pull request merged**